### DISCUSSION TITLE
Is there a way to select the tabs and create a group for them?

### DISCUSSION THREAD

**Discussion by DemDemD** [piroor/treestyletab#3338] [Q&A] [answered]
In Chrome, I can select a few tabs and create a specific group for them with name.  Is it possible to do this with TreeStyleTab in FF?
I've also notice that there's an extension SimpleTabGroups that does the job but not sure if it will work with TreeStyleTab.--Does anyone know?

**Comment by ThomasWynne** [comment, marked as answer]
https://addons.mozilla.org/en-US/firefox/addon/tst-more-tree-commands/

It looks like this extension does what you want.  If not you might want to consult [this list](https://github.com/piroor/treestyletab/blob/trunk/README.md#extensions-that-extend-tst) for other extensions.

**Reply by DemDemD** [reply]
Thanks.  I've actually found another all-in-one extension.  Been trying out "Sidebery".
